Anime-App/Feature/Anime/URL/URL.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  URL.swift
+//  Anime-App
+//
+//  Created by Tobi Adegoroye on 12/08/2023.
+//
+
+import Foundation
+
+enum UrlEndpoint {
+    case baseUrl
+    case getAnime
+    case searchAnime
+}
+
+extension UrlEndpoint {
+    var endpoint: String {
+        switch self {
+        case .baseUrl:
+            return "https://kitsu.io/api/edge"
+        case .getAnime:
+            return "/anime"
+        case .searchAnime:
+            return "/anime?filter[text]="
```

Comment:
Also do this defo work? Can you install proxyman and test it out against postman too see if the filtering text works. Cos i would expect this to fail